registry/server/apps/routes/createApps.ts

```diff
@@ -5,23 +5,39 @@ import {
 import _ from 'lodash/fp';
 
 import db from '../../db';
-import preProcessResponse from '../../services/preProcessResponse';
+import validateRequest, {
+    ValidationPairs,
+    selectBodyToValidate,
+} from '../../common/services/validateRequest';
+import preProcessResponse from '../../common/services/preProcessResponse';
 import prepareAppsToInsert from '../services/prepareAppsToInsert';
-import App, { AppBody } from '../interfaces/App';
+import App, {
+    AppBody,
+    appsBodySchema,
+} from '../interfaces/App';
 
-type CreateAppsRequestBody = Array<AppBody>;
+export type CreateAppsRequestBody = Array<AppBody>;
+
+const createAppsRequestBodySchema = appsBodySchema.min(1);
+const createAppsValidationPairs: ValidationPairs = new Map([
+    [createAppsRequestBodySchema, selectBodyToValidate],
+]);
+
+const validateAppsBeforeCreate = validateRequest(createAppsValidationPairs);
 
 const selectAppsNames = _.map<AppBody, string>(_.get('name'));
```

Comment:
Rm this function. Put it's content right in code.

Reply:
Removed